features/cron-event.feature

```diff
@@ -37,3 +37,26 @@ Feature: Manage WP Cron events
       """
       Executed a total of 1 cron event
       """
+
+  @require-wp-4.9.0
+  Scenario: Unschedule cron event
+    When I run `wp cron event schedule wp_cli_test_event_1 now hourly`
```

Comment:
Could we add a test case of multiple events as well?

Reply:
Done.